_posts/2020-06-03-project-workflow.md

```diff
@@ -0,0 +1,87 @@
+
+In this scenario, even if a different reviewer approves the pull request, it cannot move along in the process until it is approved by the reviewer who requested changes.
+
+Additionally, if the Pull Request was approved by a reviewer and a second one reviews it and requests changes, the Pull Request is not considered code reviewed until those changes are addressed.
+
+## Step 4: Approving a Pull Request and Deploying to Staging
+
+Once the Pull Request has been approved, the reviewer must deploy the relevant branch to our staging environment and add the Ready for QA label to the Pull Request, indicating it is ready for manual testing. This tells the author and the tester the pull request passed Code Review.
+
+The reviewer and the author must also ensure the Pivotal Tracker story reflects that the pull request passed code review by clicking the Deliver button in the story, putting it in an Accept / Reject state.
+
+## Step 5: Accepting and Rejecting stories
+
+After the approval of a Pull Request, it is ready to be manually tested. This is done by the Product Owner, who will make sure the implementation actually satisfies the requirements set in the story and there are no issues associated with it.
+
+**Accepting a Story**
+If the story passes manual testing, the Product Owner, removes the Ready for QA label and adds the Ready to Merge label. This means the Pull Request can be merged by the author.
+
+The Product Owner can then accept the story in Pivotal Tracker, indicating it has been successfully implemented.
+
+**Rejecting a Story**
+If the Product Owner encounters issue with the implementation of the story during manual testing, they must note these issues in the Pull Request in the form of a comment, remove the Ready for QA label and apply the Needs Revision label.
+
+They must also reject the story in Pivotal Tracker. This will put it in a Restart state and the developer must follow the process again when restarting the story and addressing the issues encountered.
+
+## What if there are issues after deploying to production?
+
+We should not be shipping bugs to production. If after a Pull Request has been merged and the story has been accepted and deployed to production an issue is found, the story **should not be changed to rejected**.
+
+At that point, a **new** user story should be created with the type set to **bug** and the issue found should be described. The team will then proceed to follow the process to fix the bug.
+
+## Pair Programming
+
+We like to use Pair Programming whenever it makes sense as a technique. If two developers work on an issue and open a Pull Request while pair programming, it is considered code reviewed and Ready for QA. Therefore, it should be labeled as such and a comment should be added to the Pull Request explaining it was opened during a pair programming session.
```

Comment:
You can link Pair Programming to Fiona's blogpost -> https://www.ombulabs.com/blog/learning/pair-programming/how-to-pair-sucessfully.html

```suggestion
We like to use [Pair Programming](https://www.ombulabs.com/blog/learning/pair-programming/how-to-pair-sucessfully.html) whenever it makes sense as a technique. If two developers work on an issue and open a Pull Request while pair programming, it is considered code reviewed and Ready for QA. Therefore, it should be labeled as such and a comment should be added to the Pull Request explaining it was opened during a pair programming session.
```